applications/ananke/doc/index.md

```diff
@@ -0,0 +1,161 @@
+/*
+Section: Ananke
+Title: Ananke
+Language: en-US
+*/
+
+# Ananke *Callback features*
+
+This application originate calls on various events
+
+## *new_voicemail*
+
+When this event is received the following fields are checked
+
+** `notify_callback_number` in corresponding vmbox
+** or `vm_notify_callback_number` in vmbox owner's document
+
+to get a number.
+
+Also the following fields are checked
+
+** `notify_callback_disabled` in vmbox
+** `vm_notify_callback_disabled` in vmbox owner's document
+
+If number was determined and notifications are not disabled, the call is being originated. The second leg is voicemail check callflow number.
+A-leg custom channel variable `Caller-ID-Number` is set being equal to `mailbox` parameter in corresponding document.
+
+A call will be attempted `attempts` times every `interval` minutes. Call timeout can be adjasted by modifying `callTimeout` option (in seconds)
```

Comment:
Is it `callTimeout`? Should be `call_timeout_s` for case and so we can tell time unit at a glance.